translations/OWNERS

```diff
@@ -1,6 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
-
-reviewers:
```

Comment:
needs rebasing on the "add sigs to root folders" commit, I think, and then bindata regenerated